specs/endpoints/food_delete.spec.js

```diff
@@ -0,0 +1,101 @@
+var shell = require('shelljs');
+var request = require("supertest");
+var app = require('../../app');
+var Food = require('../../models').Food;
+
+describe('Food delete API', () => {
+  beforeAll(() => {
+    shell.exec('npx sequelize db:create')
+  });
+  beforeEach(() => {
+    shell.exec('npx sequelize db:migrate')
+    shell.exec('npx sequelize db:seed:all')
+  });
+  afterEach(() => {
+    shell.exec('npx sequelize db:seed:undo:all')
+  });
+
+  describe('', () => {
+    test('it should delete a food successfully', () => {
+      // Below is an alternative way to create a food. Although, by utilizing this way,
+      // I am receiving the error that food cannot be created without a name, I don't understand
+      // why the given name of Cheese Puffs is not being utilized.
+
+      // const id = 83
+      // Food.create({
+      //   where: {
+      //     id: id,
+      //     name: 'Cheese Puffs',
+      //     calories: 1000
+      //   }
+      // })
+
+
+// An alternative way to create a food is to send a POST request, like below.
+//  Check the count of Food items in the database, then send a DELETE request,
+// And check the count of items in the database again.
+
+      const newFood = {
+        name: "Pringles",
+        calories: 27
+      }
+
+      return request(app).post("/api/v1/foods").send(newFood)
+      .then(response => {
+        expect(response.status).toBe(200)
+        const matchingFoods = Food.findAndCountAll({
+          where: {
+            name: "Pringles"
+          }
+        })
+          .then(foods => {
```

Comment:
Whether created manually or by post request, should be able to find the food by name and then grab the return food.id to store and send in your delete request.

Reply:
I've tried this with several different approaches. Do you have functional code that does this?